### PR TITLE
[FIX] Switch component not working.

### LINKED ISSUE #7057: [Toggle switch] Broken change event callback
The onChange event handling is broken for the Switch component. 
Toggling a switch doesn't result in the onChange event being handled, even though it's sent from the SwitchViewManager. This makes the switch rather useless.

Verified in: view.tsx, rntester\SwitchExample.js, rntester\FlatListExample

### PR DESCRIPTION
**Pull request by rectified95**
**tl; dr;** We have a typo that breaks the Switch component, and here's a fix.

Currently, the Switch component doesn't work - toggling it doesn't result in the `onChange` callback being invoked.
I debugged it and found out that deep in the `ReactNativeRenderer` the event listener for the `topChange` event in the `bubbled` phase of capture was not getting located. Tracing it all the way up through the construction of the `ViewConfig`, and then across the bridge to `OInstance` and the `remoteModuleConfig` object, I noticed that we corrupt the `customBubblingEventTypes` object that gets passed on to the `ReactNativeViewConfigRegistry` - the `phasedRegistrationNames` field contained ['captured', 'cap**bubbled**tured'], which produced a cascading error and the onChange callback was not working.

(This also affect the Picker and DatePicker, but since the core implementations are deprecated no-one has noticed.)

Fixes #7057 

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/7061)

### CODE REVIEW

**Comment by asklar** [comment]
Nice catch!
fyi looks like you ran yarn change against an old commit, make sure you run `yarn change --branch upstream/master`

**Review by asklar — APPROVED** [review]
